src/main/java/com/amazonaws/services/neptune/io/Stream.java

```diff
@@ -108,7 +110,14 @@ public void flushRecords() {
         kinesisProducer.flushSync();
     }
 
-    private static final FutureCallback<UserRecordResult> CALLBACK = new FutureCallback<UserRecordResult>() {
+    public static class UserRecordCallback implements FutureCallback<UserRecordResult> {
+
+        private final KinesisProducer kinesisProducer;
```

Comment:
kinesisProducer is unused now that flushing after successful writes has been removed. This changes to this class could be reverted.

Reply:
Agreed

Reply:
Reverted to using static callback